lib/src/main/java/com/orange/ods/compose/component/button/OdsIconToggleButton.kt

```diff
@@ -57,76 +54,44 @@ import com.orange.ods.theme.colors.OdsColors
 fun OdsIconToggleButton(
     checked: Boolean,
     onCheckedChange: (Boolean) -> Unit,
-    icon: Painter,
-    contentDescription: String,
+    painter: Painter,
```

Comment:
Should we rename this parameter to `uncheckedPainter` for clarity ?

And `checkedPainter` seems more English accurate than `painterChecked` to me.